src/lib/theme/default.ts

```diff
@@ -138,7 +138,7 @@ const theme: FlowbiteTheme = {
       },
       initials: {
         text: 'font-medium text-gray-600 dark:text-gray-300',
-        base: 'inline-flex overflow-hidden relative justify-center items-center w-10 h-10 bg-gray-100 dark:bg-gray-600',
+        base: 'inline-flex overflow-hidden relative justify-center items-center bg-gray-100 dark:bg-gray-600',
```

Comment:
This looks good, but there should be entries in the theme for default sizes right? xs, sm, etc.

Reply:
Yes, there are here: https://github.com/themesberg/flowbite-react/blob/main/src/lib/theme/default.ts#L113-L119

Reply:
Glossed over that completely, thank you!